### ISSUE TITLE
Create Tests for Admin view vs regular user view

### ISSUE THREAD

**Issue by AndrewM131** [uchicago-cs/chigame#371]

**Comment by AndrewM131** [comment]
Description: This issue will create tests that will see whether the admin view page properly displays information that a regular user shouldn't see, and test what a regular user would see.

**Comment by AndrewM131** [comment]
Update: We created two tests to test admin vs user view. Also, we implemented the user list to only be viewed by admins, and this page can be accessed on their account.

**Comment by AndrewM131** [comment]
Closing statement: In this issue, we successfully implemented tests such that we can test that an admin has a different view than a user. This was made possible by making users_list an admin-exclusive page, and this is linked in the admin account page, so only admins can access this page. If a user tries to access this page via a link, they are given a 404 page. In the future, this should redirect them somewhere else, such as their account, and a future team can implement that. 
Ishi and I worked on this together with 2 meetings throughout the sprint. I took care of labeling and answering questions, and we communicated over slack on our progress of the issue/PR.
